delfin/drivers/netapp/dataontap/netapp_handler.py

```diff
@@ -370,26 +376,35 @@ def get_disks(self, storage_id):
             constant.DISK_SPLIT_STR)
         physicals_info = self.ssh_pool.do_exec(
             constant.DISK_SHOW_PHYSICAL_COMMAND)
+        error_disk = self.ssh_pool.do_exec(
+            constant.DISK_ERROR_COMMAND
+        )
+        error_disk_list = []
+        error_disk_array = error_disk.split('\r\n')
+        for error_disk in error_disk_array[1:]:
+            error_array = error_disk.split()
+            if len(error_array) > 2:
+                error_disk_list.append(error_array[0])
         disks_map = {}
         physical_array = physicals_info.split('\r\n')
         for i in range(2, len(physical_array), 2):
             physicals_list.append(physical_array[i].split())
         for disk_str in disks_array[1:]:
-            speed = physical_type = firmware = None
```

Comment:
Why change default value from `None` to `'-'`

Reply:
No local synchronization, has been modified